unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/StepsController/BuildInWorldCompleteAction.cs

```diff
@@ -0,0 +1,65 @@
+using DCL.Models;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BuildInWorldCompleteAction
+{
+    public enum ActionType
+    {
+        MOVE = 0,
+        ROTATE = 1,
+        SCALE = 2,
+        CREATED = 3,
+        DELETED = 4
+    }
+
+    public ActionType actionType;
+    public bool isDone = true;
+    public System.Action<string, object, ActionType, bool> OnApplyValue;
```

Comment:
```suggestion
    public event System.Action<string, object, ActionType, bool> OnApplyValue;
```

Maybe we can use a custom delegate here? This is a complex callback and having more context of what the arguments do would be useful.